pop/POPAnimationTracer.mm

```diff
@@ -32,18 +32,19 @@ @implementation POPAnimationTracer
     : self->_animationState->lastTime;
 
   POPAnimationEvent *event;
+  __strong POPAnimation* animation = self->_animation;
```

Comment:
Wouldn't it be `__strong` implicitly by default? Not sure if we need to declare it as `__strong` here, but am fine with being explicit here.